tests/test/export/test.sh

```diff
@@ -39,8 +39,14 @@ rlJournalStart
 
     # 2 - (negative) format testing
     rlPhaseStartTest "Invalid format"
-        rlRun -s "tmt tests export --format weird" 2
-        rlAssertGrep "Invalid test export format" $rlRun_LOG
+        rlRun -s "tmt tests export --how weird" 2
+        if rlIsRHELLike "=8"; then
+            # RHEL-8 and Centos stream 8 usually offer an older Click package that has slightly
+            # different wording & quotes.
+            rlAssertgrep "Error: Invalid value for \"-h\" / \"--how\": invalid choice: weird. (choose from dict, nitrate, polarion, yaml)" $rlRun_LOG
+        else
+            rlAssertGrep "Error: Invalid value for '-h' / '--how': 'weird' is not one of 'dict', 'nitrate', 'polarion', 'yaml'." $rlRun_LOG
```

Comment:
Do we need to check the complete error message? Test will need another update when a new export plugin is added. What about something like this?

```suggestion
            rlAssertGrep "Error: Invalid value for.*-h.*--how" $rlRun_LOG
```

Reply:
The goal was to make sure the default list of plugins is displayed as part of the error message.

Reply:
Ok, understood.